### ISSUE TITLE
Internationalization support

### ISSUE THREAD

**Issue by vibragiel** [jointakahe/takahe#95]
It would be nice if the UI, e-mails and other user facing surface areas could be translated to other languages.

If you consider this is something worth doing, I guess the sooner the better, while there are few hardcoded copies to adapt.

**Comment by andrewgodwin** [comment]
Yup, the plan is to use Django's built-in i18n support for all that; I believe I've left all the strings in a place where they can be wrapped, it's just a matter of going through and doing so once the codebase settles down slightly.

(I have avoided it so far as I don't want to start getting people translating stuff that might get deleted next week)

**Comment by astrojuanlu** [comment]
Is the project in a moment that the strings can be considered sufficiently stabilized? And would you take PRs for the initial setup?

**Comment by shuuji3** [comment]
I'm willing to contribute to Japanese translation anytime once ready.
